client/status/viewmodel/render.go

```diff
@@ -493,15 +495,17 @@ func renderPrunerReport(t *stringbuilder.B, r *pruner.Report, fsfilter FilterFun
 	}
 
 	// global progress bar
-	progress := int(math.Round(80 * float64(completedDestroyCount) / float64(totalDestroyCount)))
-	t.Write("Progress: ")
-	t.Write("[")
-	t.Write(stringbuilder.Times("=", progress))
-	t.Write(">")
-	t.Write(stringbuilder.Times("-", 80-progress))
-	t.Write("]")
-	t.Printf(" %d/%d snapshots", completedDestroyCount, totalDestroyCount)
-	t.Newline()
+	if state != pruner.Done {
+		progress := int(math.Round(80 * float64(completedDestroyCount) / float64(totalDestroyCount)))
+		t.Write("Progress: ")
+		t.Write("[")
+		t.Write(stringbuilder.Times("=", progress))
```

Comment:
@problame any reason why this doesn't use `t.DrawBar()` like the code above?

Reply:
Just an oversight, I think. @0x3333, feel free to fix this in this PR.

Reply:
I just added the `if`(😅), but I updated it to use DrawBar. I used `changeCount` equal `1` so the progress doesn't show as stalled.